guides/v2.2/cloud/project/project-start.md

```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
 {: .bs-callout .bs-callout-info}
 In Production, each node in the three-node cluster has a `/tmp` directory that is not shared with the other nodes.
 
-## Logs {#logs}
+## Ignoring files
 
-Logs for all environments are located under the `/var/log` directory. You can access that directory by opening an SSH tunnel to the environment using the `magento-cloud environment:ssh -e <environment id>` command.
+We include a base `.gitignore` file with the {{site.data.var.ece}} project repository. See the latest [.gitignore file in the magento-cloud repository](https://github.com/magento/magento-cloud/blob/master/.gitignore). If you need to add a file that is in the ignore list, you can use the `-f` (force) option when staging a commit:
 
-For Pro, the deployment log for Staging and Production is located in `/var/log/platform/<project ID>`.
+```bash
+git add <path/filename> -f
+```
+
+## Logs
+
+The `/var/log` directory contains logs for all environments. Use the `magento-cloud ssh` command to log in to the environment to access the `/var/log` directory.
 
-Magento logs are located in the `<magento root dir>/var/log` directory.
+For Pro, the deployment log for Staging and Production is in `/var/log/platform` directory. Magento-specific logs are in the `<magento-root-dir>/var/log` directory.
 
 {:.bs-callout .bs-callout-tip}
 You can also [set up log-based Slack and email notifications]({{ page.baseurl }}/cloud/env/setup-notifications.html).
 
-## Build logs {#build-log}
+### Build logs
 
-After pushing to your environment, you can see the results of the both hooks. Logs from the build hook are redirected to the output stream of `git push`, so you can observe them in the terminal or capture them (along with error messages) with `git push > build.log 2>&1`.
+After pushing changes to your environment, you can observe the build results in the terminal or use SSH to log in and view them in the `var/log/cloud.log` file.
 
-### Deploy logs {#deploy-log}
+### Deploy logs
 
 You can review these logs via SSH into the environment. Change to the directories listed below to review the logs.
```

Comment:
You can review .... via SSH into ...
Better if it was more like the sentence on line 69 above ... use SSH to log in -- also "these logs" ==> deploy logs

Reply:
It actually SHOULD be like that. Good catch.